pywr/optimisation/platypus.py

```diff
@@ -106,7 +106,7 @@ def evaluate(self, solution):
                 ints = np.round(np.array(x[-var.integer_size:])).astype(np.int32)
                 var.set_integer_variables(ints)
 
-        run_stats = self.model.run()
+        self.run_stats = self.model.run()
```

Comment:
Won't this attribute get continuously overwritten during the optimisation?

Reply:
Yes, but this allows you to access the run statistics if you subclass the wrapper. E.g. if you want to save them somewhere.

Reply:
ah ok